scripts/budo.js

```diff
@@ -14,24 +15,24 @@ function execCmd (cmd) {
 var consts = {
   NAME: 'AFRAME',
   ENTRY: './src/index.js',
-  DIST: 'dist/aframe-core.js',
-  BUILD: 'build/aframe-core.js',
-  WATCH: '{examples,lib,src,style}/**/*',
```

Comment:
well, `src/lib/` won't get watched _unless_ it's `require`d somewhere from the entry point (`./src/index.js`), but that should be fine IMO. feel free to add `src/lib/**` though.